torchgeo/transforms/indices.py

```diff
@@ -35,6 +35,19 @@ def ndbi(swir: Tensor, nir: Tensor) -> Tensor:
     return (swir - nir) / ((swir + nir) + _EPSILON)
 
 
+def nbr(nir: Tensor, swir: Tensor) -> Tensor:
+    """Compute Normalized Burn Ratio (NBR).
+
+    Args:
+        nir: tensor containing nir band
+        swir: tensor containing swir band
+
+    Returns:
+        tensor containing computed NBR values
```

Comment:
Can you add:
```rst
.. versionadded:: 0.2
```
to the end of the docstring for these two new methods? We've trying to document when each feature is added so users don't try to use a feature and be surprised that it doesn't exist in their version of TorchGeo.